CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Enh #286: Change property `Schema::$typeMap` to constant `Schema::TYPE_MAP` (@Tigrov)
 - Bug #287: Fix `DMLQueryBuilder::insertWithReturningPks()` and `Command::insertWithReturningPks()` methods (@Tigrov)
+- Enh #292: Resolve psalm issues (@Tigrov)
```

Comment:
```suggestion
- Enh #292: Minor refactoring of `Command` and `Quoter` (@Tigrov)
```

"Resolve psalm issues" is not about public interface of package and too common description of changes. Suggest replace it to refactoring...